src/rule-transform/rule-content/get-rule-body.ts

```diff
@@ -34,3 +58,54 @@ function stripReferences({ body, markdownAST }: Args): string {
   }
   return body.substr(0, offset);
 }
+
+function addInputAspects(
+  frontmatter: AtomicRuleFrontmatter,
+  body: string,
+  supportKeys: AccessibilitySupport
+): string {
+  frontmatter.input_aspects.forEach((aspect) => {
+    if (aspect.toLowerCase() in supportKeys) {
+      body = updateAccessibilitySupportInBody(
+        body,
+        supportKeys[aspect.toLowerCase()]
+      );
+    }
+  });
+  return body;
+}
+
+function updateAccessibilitySupportInBody(
+  body: string,
+  support: string
+): string {
+  const supportIndex = body.search(/\n###\s+Accessibility\s+Support\b/m);
+  if (supportIndex === -1) {
+    console.log("no support index");
+    return body;
+  }
+  const startOfNextHeading = body.indexOf("#", supportIndex + 4);
+  if (body.substring(supportIndex, startOfNextHeading).includes(support)) {
+    // already exists, do nothing
+    return body;
+  }
+  const currentSupportText = body.substring(supportIndex, startOfNextHeading);
+  if (
+    currentSupportText.includes(
+      "There are no accessibility support issues known."
+    )
+  ) {
+    // no support issues, replace with new support issue
+    return (
+      body.substring(0, supportIndex) +
+      `\n### Accessibility Support\n\n${support}\n\n` +
```

Comment:
is it a level 3 heading? I am seeing it as 2 here https://www.w3.org/WAI/standards-guidelines/act/rules/73f2c2/ but let me know if I am looking at the wrong page